ci/no-superuser/create-no-superuser-sqlserver.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
-set -ue
+set -u
```

Comment:
In this script, we cannot set the `-e` option. This is because we implement the retry feature in this script. If we set the `-e` option, this script will exit without retrying. I added this `-e` option by mistake in a previous PR #2084.